docs/requirements.txt

```diff
@@ -1,3 +1,5 @@
 sphinx-github-changelog
 sphinxcontrib-openapi
 sphinx-notfound-page
+# 2023-08-28: pin sphinx to 7.1.2 to avoid import error in sphinx-notfound-page with sphinx 7.2.4 and notfound 0.8
+sphinx==7.1.2
```

Comment:
We can remove this when https://github.com/readthedocs/sphinx-notfound-page/commit/dcb3adc6d75ea1e5f3af599c35dbea132f17602b is released